Add benchmark for connection fields

jnak opened the pull request:
PR preview

jnak committed:
add benchmark for connection fields

Nabellaleen commented:
Nice ! I already imagine how a Github Action/Bot could display the benchmark comparison with the previous version ;)
But I think this should be for another PR ;)

jnak commented:
@Nabellaleen Thanks for reviewing! Though I can't merge it yet in the `graphene-sqlalchemy` repo because I need to send a PR once I merge https://github.com/graphql-python/graphene-sqlalchemy/pull/254 in `master`.

jnak committed:
Fix N+1 problem for one-to-many and many-to-many relationships (#254)

This optimization batches what used to be multiple SQL statements into a single SQL statement. For now, you'll have to enable the optimization via the `SQLAlchemyObjectType.Meta.connection_field_factory` (see `test_batching.py`).

jnak committed:
bump pre-commit to fix build

jnak committed:
empty commit

jnak committed:
bump pre-commit to fix build

jnak committed:
drop active support for 3.4

jnak committed:
disable benchmark for SQLAlchemy < 1.2

jnak committed:
empty commit
